h2o-genmodel/src/main/java/hex/genmodel/attributes/metrics/MojoModelMetricsBinomial.java

```diff
@@ -2,7 +2,7 @@
 
 import hex.genmodel.attributes.Table;
 
-public class MojoModelMetricsBinomial extends MojoModelMetrics {
+public class MojoModelMetricsBinomial extends MojoModelMetricsSupervised {
   
   @SerializedName("AUC")
   public double _auc;
```

Comment:
add pr_auc?

Reply:
It's in the predecessor, MojoModelMetrics, which is a predecessor of MojoModelSupervised. The inheritance copies the inheritance model of H2O's MOdelMetrics (with some exceptions).